examples/server/server.cpp

```diff
@@ -962,10 +991,9 @@ int main(int argc, char ** argv) {
         return 1;
     }
 
-    LOG_INFO("HTTP server listening", {
-        { "hostname", sparams.hostname },
-        { "port", sparams.port },
-    });
+    std::cout << std::endl;
```

Comment:
1. don't use `std::cout` , we use `(f)printf`
2. don't use `std::endl`, it's excessive. use '\n' instead :)

Reply:
@SlyEcho why is the logging mostly json? and inconstant too.

Reply:
It should be all JSON. What is inconsistent about it?

Reply:
@tobi, please don't change the logging format.